doc/man3/EVP_PKEY_size.pod

```diff
@@ -37,11 +41,26 @@ of cryptographic length is specific to the key cryptosystem.
 EVP_PKEY_security_bits() returns the number of security bits of the given
 I<pkey>, bits of security is defined in NIST SP800-57.
 
+EVP_PKEY_get_params() gets arbitrary parameters from the I<pkey>.  The
```

Comment:
Hmm - not sure we should keep this method.. The individual accessors are a lot less error prone.

Reply:
I don't see that one needs to be gone in favor of the other...  while the individual accessors are absolutely fine if all you need is one item, I suspect that the churn is quite high as soon as you need to get more than one item in one go.